runtime/src/main/java/org/corfudb/runtime/view/ManagementView.java

```diff
@@ -10,6 +12,7 @@
 import org.corfudb.protocols.wireprotocol.orchestrator.QueryResponse;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.corfudb.protocols.wireprotocol.orchestrator.QueryResponse'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

Reply:
done.